xarray/core/variable.py

```diff
@@ -1229,12 +1390,17 @@ def chunk(self, chunks=None, name=None, lock=False):
         return self.copy(deep=False)
 
     def __getitem__(self, key):
-        key = self._item_key_to_tuple(key)
-        values = self._indexable_data[key]
-        if not hasattr(values, 'ndim') or values.ndim == 0:
+        dims, index_tuple, new_order = self._broadcast_indexes(key)
+        if len(dims) > 1:
+            # returns Variable rather than IndexVariable if multi-dimensional
+            return Variable(self.dims, self.data, self._attrs, self._encoding,
+                            fastpath=True)[key]
+
+        values = self._indexable_data[index_tuple]
+        if getattr(values, 'ndim', 0) == 0:
```

Comment:
Can this be: `if getattr(values, 'ndim', 0) != 1` instead?

Reply:
Done